### DISCUSSION TITLE
Should I set Fan curve to Auto Apply?

### DISCUSSION THREAD

**Discussion by cheonmux** [seerge/g-helper#127] [Q&A] [answered]
Hello. 

First of all, thank you very much for developing such a useful program.

I am using only G-helper since I uninstalled both armory crater and myasus. (G15-2022)

The program works fine, but I don't know if I should check the auto apply box for Fan cuve.

If i checked the auto apply and the fan seems to spin a little more often... Should I check it?

I'm not modifying the fan controls, I'm just using the G-helper`s defaults setting.

And do I need to do any registry modification beforehand to change the CPU Booster?

Or can I change it in g-helper and it will be applied automatically?

Thank you.

**Comment by seerge** [comment, marked as answer]
Hello

- If you don't want any custom fan profiles - just keep it unchecked / and don't click Apply. Asus BIOS will always use default fan curve :) If you click apply it's always treated by asus bios as "custom" (even if it looks the same).
- You don't need to do registry modifications. Setting boost to off (or anything else) adjusts equivalent flag in active windows power plan

**Reply by cheonmux** [reply]
Thank you so much for your quick response!

However, I already clicked auto apply and apply fan curve, can I just turn off auto apply and do a factory reset?

**Reply by seerge** [reply]
Every time you select any performance mode bios resets any custom settings. So just re-select same mode. Or you can click "factory defaults" - it does same thing

**Reply by cheonmux** [reply]
Okay, thank you!
